setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```

Comment:
Missing newline

Reply:
oops spillover, is it needed?? @Juanlu001

Reply:
Not mandatory, but when doing `$ cat setup.cfg` you will see the last line merged with your prompt. Just aesthetic details :)

Reply:
no, i mean - i added setup.cfg half-accidentally, spillover from some other project.
do we want this file here?